get_webpage.py

```diff
@@ -21,8 +21,22 @@ def shorten_title(title):
     Returns: The last segment of the input PV address as string.
 
     """
-    number = title.rfind(':')
-    return title[number + 1:]
+    title = title.split(':')
+    rc_name = "RC"
+    value_name = ["HIGH.VAL", "LOW.VAL", "INRANGE.VAL", "ENABLED.VAL"]
+    title_list = []
+
+    if rc_name in title:
+        title_list.append(title[-3])
+        title_list.append(rc_name)
+        for name in value_name:
+            if name in title:
+                title_list.append(name)
+                title = ':'.join(title_list)
+                return title
+    else:
+        title = title[-1]
+        return title
```

Comment:
This code looks identical to that in `shorten_rc_title`. We should refactor to just use a common function instead.